sevenseconds/config/vpc.py

```diff
@@ -788,6 +788,11 @@ def if_stups_tool(ni: dict):
             for gateway in ec2c.describe_nat_gateways()['NatGateways']:
                 if gateway.get('NatGatewayAddresses', {})[0].get('AllocationId') == allocation_id:
                     return True
+
+        # KMS VPC Endpoint
```

Comment:
it's filtering by the security group name, right? Maybe just add a comment, because AWS saved 8 characters not calling it "SecurityGroups"

Reply:
right, didn't realize this was security group name when I just looked at the API reference. Will change it.